.github/workflows/docker-build.yaml

```diff
@@ -28,6 +26,15 @@ jobs:
           - linux/arm64
 
     steps:
+      # GitHub Packages requires the entire repository name to be in lowercase
+      # although the repository owner has a lowercase username, this prevents some people from running actions after forking
+      - name: Set repository and image name to lowercase
+        run: |
+          echo "IMAGE_NAME=${IMAGE_NAME_MIXED_CASE,,}" >>${GITHUB_ENV}
+          echo "FULL_IMAGE_NAME=ghcr.io/${IMAGE_NAME_MIXED_CASE,,}" >>${GITHUB_ENV}
+        env:
+          IMAGE_NAME_MIXED_CASE: '${{ github.repository }}'
```

Comment:
Let's just reuse the same env var name here.

`IMAGE_NAME: '${{ github.repository }}'`
`echo "IMAGE_NAME=${IMAGE_NAME,,}" >>${GITHUB_ENV}`

The rest LGTM!

Reply:
done - [just testing it now](https://github.com/JohnTheNerd/open-webui/actions/runs/9537871843)

Reply:
the build succeeded - resolving the conversation now